readme.md

```diff
@@ -43,6 +43,23 @@ Force it to be enabled/disabled.
 
 #### .isEnabled(): `Promise<boolean>`
 
+#### .createEmitter(milliseconds): `Function<EventEmitter>`
```

Comment:
Instead of a function, I would prefer just two getters that creates it on the first call to either and then reuses it.

```js
dotNotDisturb.on('change')
dotNotDisturb.off('change')
```

Reply:
I've updated it, let me know if this is what you had in mind.